### ISSUE TITLE
Listbox accessibility issue

### ISSUE THREAD

**Issue by m1maker** [frang75/nappgui_src#172]
Hello!

First, thank you for this great library. I have found it to be an excellent cross-platform solution for creating GUIs!

However, I wanted to bring up an accessibility issue. While this library uses native system widgets, which is beneficial for screen reader users, the listbox widget relies on a custom implementation. As a result, it is not fully accessible to screen readers. 

I can read all the list items using screen view or other methods provided by the screen reader. However, when navigating through the list with arrow keys, the screen reader does not announce the selected item.

# My recommendations:

1. Consider using native list boxes provided by the system to ensure accessibility for users with visual impairments.

2. Alternatively, you could implement the accessibility API directly.

Thank you very much for your hard work!

**Comment by a11cf0** [comment]
The table control is affected too. Maybe there are more controls like these.

**Comment by m1maker** [comment]
Here are some accessibility API related documentation links:
### Microsoft

* [Active Accessibility](https://www.codeproject.com/Articles/18290/Introduction-to-Microsoft-Active-Accessibility)

### Apple

* [Accessibility App Kit](https://developer.apple.com/documentation/appkit/accessibility)
* [Accessibility Integration](https://developer.apple.com/documentation/appkit/accessibility/integrating_accessibility_into_your_app)

### Linux

Sorry, unfortunately there is very little documentation for Linux, but I think this will still help you.

* [ATK Library](https://docs.gtk.org/atk)
* [Custom Control Implementation](https://github.com/webczat/a11ytest)





Thank you!
